modal.js

```diff
@@ -0,0 +1,208 @@
+          2015
+        </li>
+      </ul>
+  
+      <p>
+      ${item.description}
+      </p>
+      <div class="language1-inline">
+        <ul>
+          ${item.technologies
+    .map((e) => `<li class="langu">${e}</li>`)
+    .join('')}
+        </ul>
+      </div>
+      <button type="button" class="btn-two">See project</button>
+    </div>
+  </div>
+  </div>`;
+  });
+}
+createCards();
+
+const containerMobile = document.querySelector('#projects-section-mobile');
+
+function createCardsMobile() {
+  portfolioDetails.forEach((items) => {
+    containerMobile.innerHTML += `<div class="left1">
+  <div class="project-card">
+    <div class="snap1"><img src="${items.mobileImg}"></div>
+  </div>
+  <div class="left-block">
+    <h2 class="head">${items.name}</h2>
+    <div class="card-detail-box">
+      <ul>
+        <li class="card-detail canopy">${items.Tags[0]}</li>
+        <li class="card-detail option">
+          <img src="images/bullets.png" alt="bullet" class="circle" />
+          ${items.Tags[1]}
+        </li>
+        <li class="card-detail option">
+          <img src="images/bullets.png" alt="bullet" class="circle" />
+          2015
+        </li>
+      <p>
+      ${items.description}
+      </p>
+      <div class="language1-inline">
+        <ul>
+         ${items.technologies.map((e) => `<li class="langu">${e}</li>`).join('')}
+        </ul>
+      </div>
+      <button type="button" class="btn-one1">See project</button>
+    </div>
+  </div>
+  </div>`;
+
+    const seeMe1 = document.querySelectorAll('.btn-one1');
+    const seeMe2 = document.querySelectorAll('.btn-two');
+    const seeMe3 = document.querySelectorAll('.btn-one');
+
+    const popupStringLiteral = `
+  <div class="popup-container">
+      <h1>Tonic</h1>
+      <img
+        src="./images/popup/close-icon.svg"
+        alt="close image"
+        class="close-img"
+      />
+      <img src="./images/popup/frame.svg" alt="canopy" class="popup-cano" />
+
+      <img src="./images/popup/photo1.png" alt="#" class="popup-photo display-only-mobile" />
+      <img src="./images/popup/popup-desktop.png" alt="#" class="popup-photo display-only-desktop" />
+
+      <div class="text-and-tags">
+        <p class="text display-only-mobile" >
+          ${items.moreDetailsMobile}
+        </p>
+
+        <p class="display-only-desktop">${items.moreDetails} </p>
+
+        <div class="tags-live-source-envelope">
+          <img
+            src="./images/popup/mobile-tags.svg"
+            alt="mobile tags"
+            class="tags1"
+          />
+          <img
+            src="./images/popup/tags-desktop.svg"
+            alt="tags desktop"
+            class="display-on-desktop"
+          />
+
+          <div class="source-live">
+            <img src="./images/popup/see-live.png" alt="see live" />
+            <img src="./images/popup/see-source.png" alt="see source" />
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] [REQUIRED] Good Job so far for implementing changes requested by previous reviewers But there are some points that you need to fix. First, when we click/Tap on a button it triggers all the popups, we have a duplication of the popup with the same information from one object and when you click or tap on another project button we have the same from the previous project or popup. Please fix it in order to make your popup works correctly :+1: 

Check the screencast below to check :point_down: 

[Screencast from 07-26-2022 02:46:44 PM.webm](https://user-images.githubusercontent.com/6615394/181043651-47e3fd5f-4b81-41fa-b5aa-fa3e44c7002b.webm)